src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -452,26 +396,6 @@ private void installServer42IngressFlow(final InstallServer42IngressFlow command
                 command.isMultiTable());
     }
 
-    /**
-     * Processes egress flow install message.
-     *
-     * @param message command message for flow installation
-     */
-    private void doProcessEgressFlow(final CommandMessage message, String replyToTopic, Destination replyDestination)
-            throws FlowCommandException {
-        InstallEgressFlow command = (InstallEgressFlow) message.getData();
-        logger.info("Installing egress flow '{}' on switch '{}'", command.getId(), command.getSwitchId());
-
-        try {
-            installEgressFlow(command);
```

Comment:
What about `installEgressFlow`, `installEgressFlow` and `installOneSwitchFlow` methods? What about deeper handling triggered from these methods?

Reply:
These methods have been removed.